src/cljs/rems/application_list.cljs

```diff
@@ -57,6 +58,18 @@
                         :resubmitted-application}
                       (:application/todo app)))))
 
+;; could be in some util namespace, but only used here for now
+(defn- application-overdue? [application]
+  (when-let [dl (:application/deadline application)]
+    (time/after? (time/now) dl)))
```

Comment:
Doesn't moving this past deadline -check to front-end risk having the wrong time/timezone etc. in the user computer? That's actually the reason why I initially decided to have it in the back-end.

Reply:
Timezones shouldn't affect it since we use timezoned datetimes, but yeah the client clock can be wrong. However since we use the client time only for rendering warnings, I think it's ok for now.

Reply:
Ok, I guess it's fine.